<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogExport</class>
 <widget class="QDialog" name="DialogExport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export standalone...</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Location:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditLocation"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonLocation">
         <property name="minimumSize">
          <size>
           <width>45</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>45</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Type of export</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3"/>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonDesktop">
            <property name="text">
             <string>Deploy a desktop application</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>13</width>
                <height>48</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="labelDeployOS">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>OS:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="comboBoxOSDeploy">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <item>
                 <property name="text">
                  <string>Windows</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Linux</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="0" column="2" rowspan="2">
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>238</width>
                  <height>48</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="1" column="0" colspan="2">
               <widget class="QCheckBox" name="checkBoxProtect">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Protect datas</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonBrowser">
            <property name="text">
             <string>Deploy for web browser</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogExport</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogExport</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
